core/src/test/java/se/tre/freki/storage/StoreTest.java

```diff
@@ -76,27 +75,4 @@ public void testGetNameMissingAbsent() throws Exception {
     final Optional<String> missing = store.getName(missingLabelId(), LabelType.TAGV).get();
     assertFalse(missing.isPresent());
   }
-
-  @Test
-  @Ignore
-  public void testRenameIdFoundOnNewName() throws Exception {
-    final LabelId id = store.createLabel("name", LabelType.TAGK).get();
```

Comment:
This test should not be removed. It tests that a call to rename on the `Store` really renames the label behind the ID.